### PR TITLE
doc --rpc-ws-max-active-connections option

### LINKED ISSUE #631: Doc --rpc-ws-max-active-connections command line option
PR: https://github.com/hyperledger/besu/pull/2006

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Add documentation for the --rpc-ws-max-active-connection option

## Issue fixed

fixes #631 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -3035,6 +3035,34 @@ The host for Websocket WS-RPC to listen on. The default is 127.0.0.1.
...
+    BESU_RPC_WS_MAX_ACTIVE_CONNECTIONS=100
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    rpc-ws-max-active-connections="100"
+    ```
+
+The maximum number of WebSocket connections allowed for JSON-RPC. Once this limit is reached, incoming connections will be rejected. The default is 80.
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Syntax.md:3064]: Minor comment: Try and rephrase without "will".

**Review by bgravenorst — CHANGES_REQUESTED** [review]
Minor change required.

Also, I'm not sure quotes ("") are required around integers in the config file. But we seem pretty inconsistent in the docs.

**Comment by NicolasMassart** [comment]
> Minor change required.
> 
> Also, I'm not sure quotes ("") are required around integers in the config file. But we seem pretty inconsistent in the docs.

No, quotes should not be used around numbers. see https://toml.io/en/

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -3035,6 +3035,34 @@ The host for Websocket WS-RPC to listen on. The default is 127.0.0.1.
...
+=== "Environment Variable"
+
+    ```bash
+    BESU_RPC_WS_MAX_ACTIVE_CONNECTIONS=100
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    rpc-ws-max-active-connections="100"
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md:3061]: numbers must not have quotes in TOML

**Review by NicolasMassart — APPROVED** [review]

**Review by bgravenorst — APPROVED** [review]
Looks good to me

**Pull request merged**